Cvalidparen/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Instructions:
+# Run: docker build -t autograding_validparen
```

Comment:
Seems to be missing a `.` at the end, i.e. `docker build -t autograding_validparen`